internal/jobcontainers/jobcontainer.go

```diff
@@ -237,8 +302,19 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		envs = append(envs, k+"="+expanded)
 	}
 	env = append(env, envs...)
-
-	env = append(env, sandboxMountPointEnvVar+"="+c.sandboxMount)
+	env = append(env, sandboxMountPointEnvVar+"="+c.rootfsLocation)
+
+	// Add the rootfs location to PATH so you can run things from the root of the image.
+	for idx, envVar := range env {
+		ev := strings.TrimSpace(strings.ToLower(envVar))
+		if len(ev) >= 4 && strings.ToLower(ev[:4]) == "path" {
+			rootfsLoc := c.rootfsLocation
+			if ev[len(ev):] != ";" {
+				rootfsLoc = ";" + rootfsLoc
+			}
+			env[idx] = ev + rootfsLoc
```

Comment:
It also looks like we are going to implicitly lowercase the whole env var when we write it back. We should avoid that.

Reply:
Yea this was an oversight, I'm strings.Lower'ing twice, should only need it on the "path=" check

Reply:
Fixed